### PR TITLE
Fixing Travis builds

### PR DESCRIPTION
**Pull request by PhotoNomad0**
#### Describe what your pull request addresses (Please include relevant issue numbers):
- reverted inadvertent changes in babel modules and broken babel-eslint.

#### Please include detailed Test instructions for your pull request:
- should work.

#### Standard Test Instructions for PR Review Process:

- [ ] Double check unit tests that have been written
- [ ] Check for documentation for code changes
- [ ] Check that there are not inadvertent commits to tC Apps when reviewing a tC Core PR
- [ ] Checkout the branch locally and ensure that app runs as expected
  - [ ] Ensure tests pass
  - [ ] Open and watch the console for errors
  - [ ] Make sure all actions perform as expected
  - [ ] Import and Load a new Project
  - [ ] Load a tool and perform basic actions
  - [ ] Switch tools and perform basic actions
  - [ ] Switch project to an existing project
  - [ ] Load a tool and perform basic actions
  - [ ] Switch tools and perform basic actions
  - [ ] Next time reverse the order of importing after loading an existing project
- [ ] Reviewer should double check the DoD in the ISSUE, including the “spirit” of the story
- [ ] Ask Ben or Koz if you have any concerns about implementation (especially UI related)

---
This change is [Reviewable](https://reviewable.io/reviews/unfoldingword-dev/translationcore/6459)

### CODE REVIEW

**Comment by dsm-git** [comment]
[Linux build 2.0.0 (94ff83b)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-linux-x64-2.0.0-94ff83b.deb) is ready for download.
[macOS build 2.0.0 (94ff83b)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-macos-x64-2.0.0-94ff83b.dmg) is ready for download.
[Windows x64 build 2.0.0 (94ff83b)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-win-x64-2.0.0-94ff83b.setup.exe) is ready for download.
[Windows x32 build 2.0.0 (94ff83b)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-win-x32-2.0.0-94ff83b.setup.exe) is ready for download.

**Comment by dsm-git** [comment]
[Linux build 2.0.0 (93650db)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-linux-x64-2.0.0-93650db.deb) is ready for download.
[macOS build 2.0.0 (93650db)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-macos-x64-2.0.0-93650db.dmg) is ready for download.
[Windows x64 build 2.0.0 (93650db)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-win-x64-2.0.0-93650db.setup.exe) is ready for download.
[Windows x32 build 2.0.0 (93650db)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-win-x32-2.0.0-93650db.setup.exe) is ready for download.

**Comment by dsm-git** [comment]
[Linux build 2.0.0 (3c59561)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-linux-x64-2.0.0-3c59561.deb) is ready for download.
[macOS build 2.0.0 (3c59561)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-macos-x64-2.0.0-3c59561.dmg) is ready for download.
[Windows x64 build 2.0.0 (3c59561)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-win-x64-2.0.0-3c59561.setup.exe) is ready for download.
[Windows x32 build 2.0.0 (3c59561)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-win-x32-2.0.0-3c59561.setup.exe) is ready for download.

**Comment by dsm-git** [comment]
[Linux build 2.0.0 (f95c5f3)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-linux-x64-2.0.0-f95c5f3.deb) is ready for download.
[macOS build 2.0.0 (f95c5f3)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-macos-x64-2.0.0-f95c5f3.dmg) is ready for download.
[Windows x64 build 2.0.0 (f95c5f3)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-win-x64-2.0.0-f95c5f3.setup.exe) is ready for download.
[Windows x32 build 2.0.0 (f95c5f3)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-win-x32-2.0.0-f95c5f3.setup.exe) is ready for download.

**Comment by dsm-git** [comment]
[Linux build 2.0.0 (6d5e50a)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-linux-x64-2.0.0-6d5e50a.deb) is ready for download.
[macOS build 2.0.0 (6d5e50a)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-macos-x64-2.0.0-6d5e50a.dmg) is ready for download.
[Windows x64 build 2.0.0 (6d5e50a)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-win-x64-2.0.0-6d5e50a.setup.exe) is ready for download.
[Windows x32 build 2.0.0 (6d5e50a)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-win-x32-2.0.0-6d5e50a.setup.exe) is ready for download.

**Comment by dsm-git** [comment]
[Linux build 2.0.0 (85e5f1e)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-linux-x64-2.0.0-85e5f1e.deb) is ready for download.
[macOS build 2.0.0 (85e5f1e)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-macos-x64-2.0.0-85e5f1e.dmg) is ready for download.
[Windows x64 build 2.0.0 (85e5f1e)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-win-x64-2.0.0-85e5f1e.setup.exe) is ready for download.
[Windows x32 build 2.0.0 (85e5f1e)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-win-x32-2.0.0-85e5f1e.setup.exe) is ready for download.

**Comment by dsm-git** [comment]
[Linux build 2.0.0 (ea348d0)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-linux-x64-2.0.0-ea348d0.deb) is ready for download.
[macOS build 2.0.0 (ea348d0)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-macos-x64-2.0.0-ea348d0.dmg) is ready for download.
[Windows x64 build 2.0.0 (ea348d0)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-win-x64-2.0.0-ea348d0.setup.exe) is ready for download.
[Windows x32 build 2.0.0 (ea348d0)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-win-x32-2.0.0-ea348d0.setup.exe) is ready for download.

**Comment by dsm-git** [comment]
[Linux build 2.0.0 (9f3700c)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-linux-x64-2.0.0-9f3700c.deb) is ready for download.
[macOS build 2.0.0 (9f3700c)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-macos-x64-2.0.0-9f3700c.dmg) is ready for download.
[Windows x64 build 2.0.0 (9f3700c)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-win-x64-2.0.0-9f3700c.setup.exe) is ready for download.
[Windows x32 build 2.0.0 (9f3700c)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-win-x32-2.0.0-9f3700c.setup.exe) is ready for download.

**Comment by dsm-git** [comment]
[Linux build 2.0.0 (50790e0)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-linux-x64-2.0.0-50790e0.deb) is ready for download.
[macOS build 2.0.0 (50790e0)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-macos-x64-2.0.0-50790e0.dmg) is ready for download.
[Windows x64 build 2.0.0 (50790e0)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-win-x64-2.0.0-50790e0.setup.exe) is ready for download.
[Windows x32 build 2.0.0 (50790e0)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-win-x32-2.0.0-50790e0.setup.exe) is ready for download.

**Comment by dsm-git** [comment]
[Linux build 2.0.0 (b0b98f8)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-linux-x64-2.0.0-b0b98f8.deb) is ready for download.
[macOS build 2.0.0 (b0b98f8)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-macos-x64-2.0.0-b0b98f8.dmg) is ready for download.
[Windows x64 build 2.0.0 (b0b98f8)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-win-x64-2.0.0-b0b98f8.setup.exe) is ready for download.
[Windows x32 build 2.0.0 (b0b98f8)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-win-x32-2.0.0-b0b98f8.setup.exe) is ready for download.

**Comment by dsm-git** [comment]
[Linux build 2.0.0 (4f8df9c)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-linux-x64-2.0.0-4f8df9c.deb) is ready for download.
[macOS build 2.0.0 (4f8df9c)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-macos-x64-2.0.0-4f8df9c.dmg) is ready for download.
[Windows x64 build 2.0.0 (4f8df9c)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-win-x64-2.0.0-4f8df9c.setup.exe) is ready for download.
[Windows x32 build 2.0.0 (4f8df9c)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-win-x32-2.0.0-4f8df9c.setup.exe) is ready for download.

**Comment by dsm-git** [comment]
[Linux build 2.0.0 (fa2533d)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-linux-x64-2.0.0-fa2533d.deb) is ready for download.
[macOS build 2.0.0 (fa2533d)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-macos-x64-2.0.0-fa2533d.dmg) is ready for download.
[Windows x64 build 2.0.0 (fa2533d)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-win-x64-2.0.0-fa2533d.setup.exe) is ready for download.
[Windows x32 build 2.0.0 (fa2533d)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-win-x32-2.0.0-fa2533d.setup.exe) is ready for download.

**Comment by dsm-git** [comment]
[Linux build 2.0.0 (912ad02)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-linux-x64-2.0.0-912ad02.deb) is ready for download.
[macOS build 2.0.0 (912ad02)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-macos-x64-2.0.0-912ad02.dmg) is ready for download.
[Windows x64 build 2.0.0 (912ad02)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-win-x64-2.0.0-912ad02.setup.exe) is ready for download.
[Windows x32 build 2.0.0 (912ad02)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-win-x32-2.0.0-912ad02.setup.exe) is ready for download.

**Comment by dsm-git** [comment]
[Linux build 2.0.0 (306f8e3)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-linux-x64-2.0.0-306f8e3.deb) is ready for download.
[macOS build 2.0.0 (306f8e3)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-macos-x64-2.0.0-306f8e3.dmg) is ready for download.
[Windows x64 build 2.0.0 (306f8e3)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-win-x64-2.0.0-306f8e3.setup.exe) is ready for download.
[Windows x32 build 2.0.0 (306f8e3)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-win-x32-2.0.0-306f8e3.setup.exe) is ready for download.

File: package.json [hunk 1]
```diff
@@ -17,8 +17,8 @@
     "npm-clean-up": "rm -rf node_modules/ && rm -rf tC_apps/*/node_modules && rm package-lock.json && npm i",
     "lint": "eslint ./src",
     "lint:fix": "eslint ./src --fix",
-    "test": "eslint ./src && jest",
-    "ci-test": "eslint ./src && jest --coverage",
+    "test": "eslint ./src && jest --version && jest",
```
Comment (mannycolon) [inline package.json]: Should we remove --version?

File: package.json [hunk 2]
```diff
@@ -17,8 +17,8 @@
...
+    "ci-test": "eslint ./src && jest --version && jest --coverage",
```
Comment (mannycolon) [inline package.json]: ibid

File: src/js/pages/app.js [hunk 3]
```diff
@@ -33,6 +33,7 @@ import { APP_VERSION, LOG_FILES_PATH } from '../common/constants';
 const version = `v${APP_VERSION} (${process.env.BUILD})`;
 injectFileLogging(LOG_FILES_PATH, version);
 console.log('SYSTEM INFO:\n' + getOsInfoStr());
```
Comment (mannycolon) [inline src/js/pages/app.js]: Why do we need all these console logs?

File: src/main.js [hunk 4]
```diff
@@ -26,6 +26,8 @@ let splashScreen;
 const version = `v${p.version} (${process.env.BUILD})`;
 injectFileLogging(path.join(ospath.home(), 'translationCore', 'logs'), version);
 
+console.log('process.versions: ' + JSON.stringify(process.versions));
```
Comment (mannycolon) [inline src/main.js]: More unnecessary console logs. Please remove

**Review by mannycolon — CHANGES_REQUESTED** [review]

**Comment by dsm-git** [comment]
[Linux build 2.0.0 (e38bad1)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-linux-x64-2.0.0-e38bad1.deb) is ready for download.
[macOS build 2.0.0 (e38bad1)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-macos-x64-2.0.0-e38bad1.dmg) is ready for download.
[Windows x64 build 2.0.0 (e38bad1)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-win-x64-2.0.0-e38bad1.setup.exe) is ready for download.
[Windows x32 build 2.0.0 (e38bad1)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-win-x32-2.0.0-e38bad1.setup.exe) is ready for download.

**Review by PhotoNomad0 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/unfoldingword-dev/translationcore/6459)* status: 0 of 11 files reviewed, 4 unresolved discussions (waiting on @mannycolon)

___
*[package.json, line 20 at r1](https://reviewable.io/reviews/unfoldingword-dev/translationcore/6459#-Lp__Ez3Iks0kbx67fNX-r1-20:-Lpddktm1xbEO_jW7y0A:b-896fix) ([raw file](https://github.com/unfoldingword-dev/translationcore/blob/e38bad1d1f7b6f87e7b5ef9f3b0d415cfba38b00/package.json#L20)):*
<details><summary><i>Previously, mannycolon (Manny Colon) wrote…</i></summary><blockquote>

Should we remove --version? 
</blockquote></details>

Done.
___
*[package.json, line 21 at r1](https://reviewable.io/reviews/unfoldingword-dev/translationcore/6459#-Lp__Ez3Iks0kbx67fNX-r1-21:-LpddiCvEVlDODCScQEa:b-896fix) ([raw file](https://github.com/unfoldingword-dev/translationcore/blob/e38bad1d1f7b6f87e7b5ef9f3b0d415cfba38b00/package.json#L21)):*
<details><summary><i>Previously, mannycolon (Manny Colon) wrote…</i></summary><blockquote>

ibid
</blockquote></details>

Done.
___
*[src/main.js, line 29 at r1](https://reviewable.io/reviews/unfoldingword-dev/translationcore/6459#-LpcDeZNgrmz5N-wFmiF-r1-29:-Lpddn0qA_bWNqTA7l5D:b-896fix) ([raw file](https://github.com/unfoldingword-dev/translationcore/blob/e38bad1d1f7b6f87e7b5ef9f3b0d415cfba38b00/src/main.js#L29)):*
<details><summary><i>Previously, mannycolon (Manny Colon) wrote…</i></summary><blockquote>

More unnecessary console logs. Please remove
</blockquote></details>

Done.
___
*[src/js/pages/app.js, line 35 at r1](https://reviewable.io/reviews/unfoldingword-dev/translationcore/6459#-LpcPvXzL3rwZzSFtLIS-r1-35:-LpddnYgC8tLLBQvJJo_:b-896fix) ([raw file](https://github.com/unfoldingword-dev/translationcore/blob/e38bad1d1f7b6f87e7b5ef9f3b0d415cfba38b00/src/js/pages/app.js#L35)):*
<details><summary><i>Previously, mannycolon (Manny Colon) wrote…</i></summary><blockquote>

Why do we need all these console logs?
</blockquote></details>

Done.

**Review by mannycolon — APPROVED** [review]

**Comment by dsm-git** [comment]
[Linux build 2.0.0 (18bda92)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-linux-x64-2.0.0-18bda92.deb) is ready for download.
[macOS build 2.0.0 (18bda92)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-macos-x64-2.0.0-18bda92.dmg) is ready for download.
[Windows x64 build 2.0.0 (18bda92)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-win-x64-2.0.0-18bda92.setup.exe) is ready for download.
[Windows x32 build 2.0.0 (18bda92)](https://tc-builds.door43.org/build/bugfix-mcleanb-travis-testing/translationCore-win-x32-2.0.0-18bda92.setup.exe) is ready for download.

**Pull request merged**